settings.gradle.kts

```diff
@@ -60,6 +62,9 @@ buildscript {
             // forgive for bc and bbc
             ("(ch/tutteli/atrium/api/(fluent|infix)/en_GB/" +
                 or(
+                    "IterableContainsInOrderOnly.*Spec",
```

Comment:
if it is not too complicated to narrow the scope a bit, then please do so. If basically every test case is failing, then you can keep it this way

Reply:
sure will check it, managed to narrow it to `IterableContainsInOrderOnly.*error cases `